view/frontend/web/instantsearch.js

```diff
@@ -414,6 +414,27 @@ requirejs(['algoliaBundle'], function(algoliaBundle) {
 				addWidget(search, widgetType, widgetConfig);
 			}
 		});
+
+		if (algoliaConfig.analytics.enabled) {
+			if (typeof algoliaAnalyticsPushFunction != 'function') {
+				var algoliaAnalyticsPushFunction = function (formattedParameters, state, results) {
+					var trackedUrl = '/catalogsearch/result/?q=' + state.query + '&' + formattedParameters + '&numberOfHits=' + results.nbHits;
+
+					// Universal Analytics
+					if (typeof window.ga != 'undefined') {
+						window.ga('set', 'page', trackedUrl);
+						window.ga('send', 'pageView');
+					}
+				};
+			}
+
+			addWidget(search, 'analytics', {
```

Comment:
Hey @julienbourdeau,

could you add the analytics widget to `allWidgetConfiguration` object so it's modifiable with the hook? The same way as it's done on M1: https://github.com/algolia/algoliasearch-magento/blob/develop/js/algoliasearch/instantsearch.js#L409-L437

Thanks!